src/components/BurgerMenu/burger-menu.scss

```diff
@@ -0,0 +1,145 @@
+
+@import 'globals';
+
+$menu-breakpoint: map-get($breakpoints, 'navigation-menu');
+
+.burger-wrapper {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  cursor: pointer;
+
+  @include media('>=#{$menu-breakpoint}') {
+    display: none;
+  }
+}
+
+.burger-wrapper > span {
+  position: inherit;
+  right: 2em;
+  transform: translateY(-50%);
+}
+
+.burger-menu {
+  position: absolute;
+  right: 0;
+  width: 16px;
+  height: 16px;
+  transform: translateY(-50%);
+}
+
+.burger-menu span {
+  position: absolute;
+  width: 100%;
+  height: 2px;
+  background-color: color('black');
+
+  &:first-child {
+    top: 0;
+  }
+
+  &:nth-child(2) {
+    top: 50%;
+    transform: translateY(-50%);
+  }
+
+  &:last-child {
+    bottom: 0;
+  }
+}
+
+.burger-wrapper.closed .burger-menu span {
```

Comment:
Later: can we make this it's own component, with a state `active` or not? And style that way?
It looks awesome though!